modules/admin_manual/pages/configuration/server/security_setup_warnings.adoc

```diff
@@ -41,6 +41,16 @@ Transactional file locking is disabled, this might lead to issues with race cond
 Please see xref:configuration/files/files_locking_transactional.adoc[Transactional File Locking] 
 for how to correctly configure your environment for transactional file locking.
 
+== Background Jobs
+
+```
+We recommend to enable system cron as any other cron method has possible performance and reliability implications.
```

Comment:
The "recommend text is redunant, delete the line below.
To mark the current line as important, line you have alternatives. Remove the backticks on top and below and write: NOTE: text, text, text
Improve the sentence itself like:
`We recommend to enable system cron as any other cron method...` -->
`ownCloud recommends to enable system cron, as any other cron method` (pls also see the comma making the text better readble)
Test it by making a local build